src/main/webapp/startup.js

```diff
@@ -1,21 +1,25 @@
 import Renderer from './Renderer.js';
+import addEventToLocalStorage from './popup.js'
 
 // Events array for testing
 const events = [
     {
         "name": "Breakfast",
         "location": "Zurich",
-        "time": "9:30AM - 10:30AM"
+        "startTime": "9:30AM",
```

Comment:
nit: Doesn't have to be done here but should dates and times not be correctly typed? Not strings? Also is there duplication here with time containing startTime and endTime. What if time is  "9:30AM - 10:30AM" but startTime is  "11:30AM" and endTime is "2:30PM

Reply:
Can you please explain what you mean with the duplication?

Reply:
I think normally we should get the text from the input field and then convert it to a Date object. This is what the Maps API is accepting!

Reply:
Sure. We can keep this in mind and modify it when we'll connect it to the Maps Api. I'll add a TODO